samples/javascript_nodejs/51.cafe-bot/dialogs/shared/prompts/getLocDateTimePartySize.js

```diff
@@ -205,7 +205,7 @@ module.exports = {
             }
         }
         /**
-         * Over ride repromptDialog.
+         * Over ride re-prompt Dialog.
```

Comment:
I believe this line was making a reference to the name of the dialog [here](https://github.com/southworkscom/BotBuilder-Samples/blob/29a7116ebc98b7421b644d21f355a71fbbaab750/samples/javascript_nodejs/51.cafe-bot/dialogs/shared/prompts/getLocDateTimePartySize.js#L214). If so, I dont think it should have been modified.

Reply:
You're right, I reverted the change @VictorGrycuk